packages/app-builder/src/components/Scenario/AstBuilder/AstBuilderNode/AggregationEdit/Modal.tsx

```diff
@@ -207,7 +207,7 @@ export function AggregationEditModal({
     <ModalV2.Root open={open} setOpen={onOpenChange}>
       <AggregationEditModalContext.Provider value={editAgregation}>
         {children}
-        <ModalV2.Content size="large">
+        <ModalV2.Content size="large" unmountOnHide>
```

Comment:
Required to ensure we refresh at modal opening the initial data (without that, data from one edit to the other can leak: horrible when it's two different nodes.

Using "local modals" would solve this issue by design